### PR TITLE
Add code files src and dist folders

### PR DESCRIPTION
**Pull request by Kaiserabbas**
Hi Dear Reviewers, Welcome to my pull request for TodoList(interactive lists)

Added a new JavaScript file and import it as a module:
Add event listener to the checkbox (change).
Updated items object's value for completed key upon user actions.
Implemented a function for the "Clear all completed" button.
Stored the updated array of items in local storage, so the user gets the correct list of values after the page reloads).
Previous pull request

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    />
+    <title>Wbpack Exercise</title>
+  </head>
+  <body>
+    <!-- <div id="todo-list"></div> -->
+
+      <div id="todo-list" class="todo-list"></div>
+    </div>
+
```
Comment (Danny-imo) [inline src/index.html]: - [ ] I can see that you have done well to implement and generate your HTML dynamically with Javascript, but on line 16 of your code you have a div that is not opened at any point, I suggest you remove it as it corresponds with HTML best practices
- [ ] Please remove all commented code to keep your codes clean

File: src/modules/print.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    tasks.push(task);
+    updateIndexes();
+    saveTasks();
+  }
+
+  static removeTask(index) {
+    // if (index < 1 || index > tasks.length) {
+    //   return;
+    // }
+    tasks.splice(index - 1, 1);
```
Comment (Danny-imo) [inline src/modules/print.js]: - [ ] As per JavaScript Best practices please remove all commented old pieces of code

**Review by Danny-imo — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas  👏,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious


## Highlights
Project setup correctly ✔️
App is working as expected ✔️
Apply DRY, KISS and YAGNI ✔️

## Required Changes ♻️
Great Job So far, but there are still some issues to be addressed
Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.html (hunk 1 above)
Comment (Kaiserabbas) [reply src/index.html]: Okay Noted. Thanks

File: src/modules/print.js (hunk 2 above)
Comment (Kaiserabbas) [reply src/modules/print.js]: Thank you for the suggesstion.

**Comment by Kaiserabbas** [comment]
@Danny-imo Thank you for the suggestions and issues raised. I have fixed all. 
- Unnecessary piece of code removed from index.html
- Unused code removed from print.js

**Review by Danny-imo — APPROVED** [review]
Hi @Kaiserabbas  👏,

# Status: APROVED🟢
There is nothing else to say than, its time to merge 👏

Highlights
Project setup correctly ✔️
Apply Requested Changes✔️
Used ES6 modules to write modular JavaScript ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**